milner_desvars/diff-drive.md

```diff
@@ -0,0 +1,5 @@
+# how a differential-drive robot works and how to control it
+
+A differential-drive robot is a wheeled robot with two controllable wheels, this wheels only can turn by their axis, so rotation of the robot is accomplished by rotating the wheels in opposite directions.
+
+You can control the robot by controlling the velocity and direction in which the left and right wheels turn, you can use control theory to move the robot by giving the general velocity and direction you want it to move, and by that you calculate the velocities of both wheels to accomplish that objective.
```

Comment:
The information is not enough to actually control a real robot. Please investigate about how to model the movement and the equations used to obtain the differential speed of each wheel based on the desired linear and angular speeds for the robot.